python/triton/runtime/interpreter.py

```diff
@@ -1010,6 +1087,7 @@ def _patch_lang(fn):
             _patch_builtin(lang.math, interpreter_builder)
         _patch_lang_tensor(lang.tensor)
         _patch_lang_core(lang)
+    _patch_builtin(tl.core._experimental_tensor_descriptor, interpreter_builder)
```

Comment:
@peterbell10 I thought we are going to support it only after the `experimental` prefix has been removed, can you confirm?

Reply:
We're close to removing the experimental prefix, just want to get a bit of feedback first from kernel devs. Some of the feedback was wanting interpreter support for debug-ability.

Reply:
```suggestion
```
Two minor issues:

1. I don't understand why this line is required. Can you justify? 
2. Why not enable other tma tests under the cuda directory?

Reply:
1. `desc.store(...)` is a builtin, so we need to patch it to pass in the ir builder.
2. Oops, added the missing marks.

Reply:
I get it now. Should we patch `_experimental_tensor_descriptor_base` or `_experimental_tensor_descriptor`?

Reply:
I tried patching `_base` instead and that seems to be working. I had thought that patching base wouldn't effect the sub-class, but this seems to work well.